modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/util/ObjectCodeEditorUtil.java

```diff
@@ -132,24 +132,19 @@ public enum DDMExpressionOperator {
 				"used-to-indicate-a-dependent-relationship",
 			"and"),
 		DIVIDED_BY(
-			"field_name / field_name2",
-			"divide-one-numeric-field-by-another-to-create-an-expression",
+			"/", "divide-one-numeric-field-by-another-to-create-an-expression",
 			"divided-by"),
 		MINUS(
-			"field_name - field_name2",
+			"-",
 			"subtract-numeric-fields-from-one-another-to-create-an-expression",
 			"minus"),
 		OR(
 			"OR",
 			"this-is-a-type-of-coordinating-conjunction-that-indicates-an-" +
 				"independent-relationship",
 			"or"),
-		PLUS(
-			"field_name + field_name2",
-			"add-numeric-fields-to-create-an-expression", "plus"),
-		TIMES(
-			"field_name * field_name2",
-			"multiply-numeric-fields-to-create-an-expression", "times");
+		PLUS("+", "add-numeric-fields-to-create-an-expression", "plus"),
+		TIMES("*", "multiply-numeric-fields-to-create-an-expression", "times");
```

Comment:
@Artartur, this fix does not seem to be the correct one for the described issue.

The content `field_name + field_name2` helps the user to understand how to build the expression with the `PLUS` operator, for example. Same thing for the other operators.

The most appropriate fix would be to handle the error that is thrown when accessing the custom object entries page.

Thanks!